Makefile

```diff
@@ -6,7 +6,7 @@ FILTER = racket infra/filter.rkt
 
 setup:
 	raco pkg install --skip-installed -n fpbench --auto generic-flonum
-	raco make main.rkt export.rkt transform.rkt toolserver.rkt evaluate.rkt 
+	raco make fpbench.rkt export.rkt transform.rkt toolserver.rkt evaluate.rkt 
```

Comment:
I think main.rkt and fpbench.rkt should be built?

Reply:
Will do